x-pack/plugins/fleet/public/constants/page_paths.ts

```diff
@@ -78,6 +79,7 @@ export const FLEET_ROUTING_PATHS = {
   data_streams: '/data-streams',
   settings: '/settings',
   settings_create_fleet_server_hosts: '/settings/create-fleet-server-hosts',
+  settings_quick_create_fleet_server_hosts: '/settings/quick-create-fleet-server-hosts',
```

Comment:
Do we need a new route or can we reuse `settings/create-fleet-server-hosts` to render the `FleetServerHostsFlyout` in serverless

Reply:
Ah, just posted a comment with that question 🙂 I'm not sure  yet what's cleaner.

`settings/create-fleet-server-hosts` [renders](https://github.com/elastic/kibana/blob/3080f8ca81c4f0ad0eeb090ffebb4e6f25af6044/x-pack/plugins/fleet/public/applications/fleet/sections/settings/index.tsx#L120) `FleetServerFlyout` while `settings/edit-fleet-server-hosts` renders `FleetServerHostsFlyout`, which is the component we want to reuse. So I thought the options were:
- introduce serverless logic around `settings/create-fleet-server-hosts` to decide which component to render
- introduce logic around `settings/edit-fleet-server-hosts` (don't like this one as the route name doesn't make sense)
- new route

I can try to see what the first option looks like if we don't want a new route.

Reply:
OK, testing done - you're right, it's cleaner to introduce serverless logic around `settings/create-fleet-server-hosts` 🙂 I pushed a commit.